assets/openobserve/Dockerfile

```
@@ -0,0 +1,24 @@
+
+FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
+
+COPY --from=0 /openobserve /openobserve
+COPY --from=0 /lib/x86_64-linux-gnu/liblzma.so.5 /lib/x86_64-linux-gnu/liblzma.so.5
+COPY --from=0 /lib/x86_64-linux-gnu/libz.so.1 /lib/x86_64-linux-gnu/libz.so.1
+
+ENV ZO_ROOT_USER_EMAIL="root@clpbench.com"
+ENV ZO_ROOT_USER_PASSWORD="password"
+
+# Install necessary packages
+RUN apt-get update \
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)